tools/image_processing/imagej2/imagej2_noise/imagej2_noise.py

```diff
@@ -0,0 +1,100 @@
+# Galaxy wrapper for use with Imagej2 by Greg Von Kuster
+"""
```

Comment:
@gregvonkuster this is just a suggestion. Imho, this is always a trade-off between readability and produced overhead. If you can avoid extra python files go for it, as long as the readability does not degrade too much.